test/utils/Permit2Helper.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma abicoder v2;
```

Comment:
Long time no see the `abicoder` declaration since we have already migrated to v0.8 for a while 🤣

Reply:
Indeed! After being around v0.8 for a while, now I think v0.7 is really out of date.